stablehlo/testdata/asinh_shape_float16_20_20.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN-DISABLED: stablehlo-opt -inline %s | stablehlo-translate --interpret
+// RUN-DISABLED(#1278): stablehlo-opt -inline %s | stablehlo-translate --interpret
```

Comment:
Let's factor the change that adds `#1278` out of this PR, so that it focuses on just one thing?

Reply:
Sure, created #1597